### ISSUE TITLE
[Feature Request] Add Page to Display All Hashtags on PixelFed Instance

### ISSUE THREAD

**Issue by TenviLi** [pixelfed/pixelfed#5312]
Currently, there is no dedicated page that displays all the hashtags used across the entire PixelFed instance. This makes it somewhat inconvenient for users to discover and explore content based on specific hashtags.

Having a centralized page showcasing all hashtags would allow users to easily navigate and find relevant content, fostering better engagement within the community. It could also encourage users to utilize hashtags more effectively, enhancing content discovery.

**Comment by witten** [comment]
I agree, this would be super helpful. On my Pixelfed instance, I actually resorted to writing some custom CSS that displays an example hashtag next to the search box.. just to give users an idea of which hashtags might be available. But a built-in page listing all hashtags or displaying a tag cloud would be way better.
